facets_dive/README.md

```diff
@@ -82,7 +82,7 @@ You can set these with the `spriteImageWidth` and `spriteImageHeight` properties
 
 ### Embedding Dive in Your Page
 
-Provided you've loaded the `<facets-dive>` Polymer element, you can insert an instance into your page like so.
+The project is build with [Bazel](https://www.bazel.build/). Make sure you have installed Bazel and run the Bazel BUILD files prior to proceeding. Provided you've loaded the `<facets-dive>` Polymer element, you can insert an instance into your page like so.
```

Comment:
"The project is build with" -> "The project is built with"

Also, please put each sentence on its own line.